docs/TESTING.md

```diff
@@ -1,16 +1,10 @@
 ## Tests
 
-Run `mix phx.server` to start a server in one window, then open a
-separate window and run `npm test` from the main folder. This will execute
-the following scripts in succession:
-
-* `mix test` — Phoenix tests
+* `mix test` — Elixir tests
```

Comment:
@digitalcora We should also mention `test:jest`, which runs more UI-focused frontent tests, as well as test coverage.

Reply:
The listed command `npm run test:js` runs both the Mocha and Jest tests. (but, this is only a reformatting of the info that was already here; should it be changed?)

Reply:
on second thought NVM

Reply:
Ok, then if your approval stands, I will merge this 😃